ase/src/main/java/com/ase/ase/rest/controller/BasicCovidController.java

```diff
@@ -38,7 +36,23 @@
                 new HospitalSituationPerDateDto(provinceId, Arrays.asList(
                         new HospitalSituationPerDate("10.10.2020", 12, 10),
                         new HospitalSituationPerDate("12.10.2020", 12, 12))));
+
     }
 
+    @CrossOrigin
+    @GetMapping(value = "/tableData/{province-id}", produces = "application/json")
```

Comment:
Would be nice to also have this method for all provines. I.e. /tableData returns something like {"items": [{"id": "1", "name": "Burgenland", "data": [...]}, {"id": "2", ...}]}

Reply:
Ok this is maybe my fault! I told her to call the data per province and display it, so we can filter the table by province. If we want to have howl austria displayed in the table it is true that we need to change it. We need to define it more deeply.